src/pages/Wallet/AddressesPage.tsx

```diff
@@ -93,11 +93,13 @@ const AddressesPage = () => {
         {sortAddressList(addresses).map((address) => {
           return (
             <TableRow
+              role="row"
```

Comment:
I find some `TableRow` instances without the `role="row"`. Is this expected?

It'd be nice if we could define styled-components components with some default attrs, but that would require creating an actual React component... :/

For example, look at how repeated the `role="cell" tabIndex={0}` are in this screenshot...

<details><summary>Click to see scrshot</summary>

<img width="442" alt="image" src="https://user-images.githubusercontent.com/1579899/179357390-c7e14506-174e-4928-851d-5c441e0542f6.png">

</details>

Reply:
When running the screenreader it was re-reading a lot of text when a role was applied on top of other roles

We should maybe peek at those large react component libraries which can be extended...